makefiles/pseudomodules.inc.mk

```
@@ -73,6 +73,7 @@ PSEUDOMODULES += mpu_noexec_ram
 PSEUDOMODULES += nanocoap_%
 PSEUDOMODULES += netdev_default
 PSEUDOMODULES += netdev_ieee802154_%
+PSEUDOMODULES += netdev_link_up
```

```suggestion
PSEUDOMODULES += stm32_eth_link_up
```

seems to me the more proper name now (also please order this then in alphabetically, don't just take the suggestion).